src/patternfly/components/Progress/examples/Progress.md

```diff
@@ -240,3 +267,4 @@ If this component is describing the loading progress of a particular region of a
 | `.pf-m-warning` | `.pf-c-progress` |  Changes the appearance of the progess component to indicate a warning state. |
 | `.pf-m-danger` | `.pf-c-progress` |  Changes the appearance of the progess component to indicate a danger (failure) state. |
 | `.pf-m-truncate` | `.pf-c-progress__description` | Modifies the description to display a single line and truncate any overflow text with ellipses. |
+| `.pf-m-fixed-width` | `.pf-c-progress.pf-m-outside .pf-c-progress__measure` | Modifies the measure element to be a fixed width that will hold 0-100%. Overridable by setting `--pf-c-progress__measure--m-fixed-width--Width`. |
```

Comment:
in the insights project i was in, there was some usage data that went to +1000%, maybe you could update this to support up to 1000% ?

Reply:
I'm trying to target the most common use case, and the component's default max value is 100

<img width="889" alt="Screen Shot 2020-12-02 at 1 56 45 PM" src="https://user-images.githubusercontent.com/35148959/100924743-a016db00-34a6-11eb-973b-ea1cd5f7fd35.png">

We can support 1000 by default, though that would mean that for anyone using a 0-100% scale, there will be extra space to accommodate 1000%. I see that as an edge case where a user should update `--pf-c-progress__measure--m-fixed-width--Width` - WDYT?

Although I do think I might add another example with a non-standard value that shows how to use the `--pf-c-progress__measure--m-fixed-width--Width` var.

Reply:
Yep if you could add an example that would be good